posthog/models/organization.py

```diff
@@ -176,34 +186,37 @@ def _billing_plan_details(self) -> tuple[Optional[str], Optional[str]]:
                 return (license.plan, "ee")
         return (None, None)
 
-    def update_available_features(self) -> list[Union[AvailableFeature, str]]:
-        """Updates field `available_features`. Does not `save()`."""
+    def update_available_product_features(self) -> list[ProductFeature]:
+        """Updates field `available_product_features`. Does not `save()`."""
         if is_cloud() or self.usage:
-            # Since billing V2 we just use the available features which are updated when the billing service is called
-            return self.available_features
+            # Since billing V2 we just use the field which is updated when the billing service is called
+            return self.available_product_features  # type: ignore
 
         try:
             from ee.models.license import License
         except ImportError:
-            self.available_features = []
+            self.available_product_features = []
             return []
 
-        self.available_features = []
+        self.available_product_features = []
 
         # Self hosted legacy license so we just sync the license features
         # Demo gets all features
         if settings.DEMO or "generate_demo_data" in sys.argv[1:2]:
-            self.available_features = License.PLANS.get(License.ENTERPRISE_PLAN, [])
+            features = License.PLANS.get(License.ENTERPRISE_PLAN, [])
+            self.available_product_features = [{"key": feature, "name": feature} for feature in features]
```

Comment:
Is the name always supposed to be the key? Or a more human-readable version of it?

Reply:
Yeah, the name is normally a more human readable version. I can parse out the features values to look a bit nicer.